test/helpers.bash

```diff
@@ -9,10 +9,14 @@ TESTDATA="${INTEGRATION_ROOT}/testdata"
 # Root directory of the repository.
 OCID_ROOT=${OCID_ROOT:-$(cd "$INTEGRATION_ROOT/../.."; pwd -P)}
 
+# Installation directory for Go binaries
+GOPATH=${GOPATH:-${OCID_ROOT}/cri-o/_output}
+GOBIN=${GOBIN:-${GOPATH}/bin}
```

Comment:
This breaks if `GOPATH` has more than one element in it.  Try `GOBIN=${GOBIN:-${GOPATH%%:*}/bin}` instead?